### PR TITLE
Update target_integrator.rb

### PR DESCRIPTION
**Pull request by thuongha-pizzahut**

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Update target_integrator.rb  
  [thuongha-pizzahut](https://github.com/thuongha-pizzahut)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: lib/cocoapods/installer/user_project_integrator/target_integrator.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ class TargetIntegrator
 
         # @return [Integer] the maximum number of input and output paths to use for a script phase
         #
-        MAX_INPUT_OUTPUT_PATHS = 1000
+        MAX_INPUT_OUTPUT_PATHS = 10000
```
Comment (dnkoutso) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: I don't see much of a problem with this but why not use Xcode 10 and `xcfilelists` which dont have this problem?

**Comment by amorde** [comment]
this was added as part of https://github.com/CocoaPods/CocoaPods/issues/7362 - will this still prevent that issue from occurring?

**Comment by dnkoutso** [comment]
User can also use `disable_input_output_paths` flag as well.

I am inclined to close this PR.

**Pull request closed without merging**